Custom-Video-Player/script.js

```diff
@@ -0,0 +1,41 @@
+const toggle = document.querySelector(".toggle");
+const video = document.querySelector(".player__video");
+const progressFill = document.querySelector(".progress__fill");
+const progress = document.querySelector(".progress");
+const playerSlider = document.querySelectorAll(".player__slider");
```

Comment:
If a variable using only once in the code, not necessary to create it.

Reply:
fixed